puppet/modules/jenkins_job_builder/files/centos.org/pipelines/test/nightlyTest.groovy

```diff
@@ -0,0 +1,41 @@
+pipeline {
+    agent { label 'foreman' }
+
+    stages {
+        stage('Setup Environment') {
+            steps {
+                deleteDir()
+                git url: 'https://github.com/theforeman/forklift.git'
+            }
+        }
+        stage('Provision Node') {
+            steps {
+                provision()
+            }
+        }
+        stage('Install Pipeline Requirements') {
+            steps {
+                sh "cp 'foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible/provision-duffy-test.yml' 'playbooks/provision-duffy-test.yml'"
```

Comment:
I take it this is for convenience? I think every job of this type will need to perform this action. Any way to just combine it? I could also see this living in forklift instead of the CI repo as it's useful.

Reply:
I'd be 100% fine with moving it to forklift. I just felt awkward about putting everything in forklift for this job.